src/components/NavBar/NavBarModal.module.scss

```diff
@@ -0,0 +1,9 @@
+.transparentModal {
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0);
+    height: 100%;
+    width: 100%;
```

Comment:
Since this class is injected into a portal, it's worth to try using `100vh` and `100vw` instead of `%` and see if it takes the entire screen. Tell me if it worked.

Reply:
It works.